src/test/ui/alloc-error/alloc-error-handler-bad-signature-3.stderr

```diff
@@ -17,8 +17,10 @@ LL | fn oom() -> ! {
    = note: this error originates in the attribute macro `alloc_error_handler` (in Nightly builds, run with -Z macro-backtrace for more info)
 help: remove the extra argument
    |
-LL | fn oom() -> !() {
-   |              ++
+LL - fn oom() -> ! {
+LL -     loop {}
+LL - }
```

Comment:
This suggestion is weird too. Any idea why?

Reply:
I think this is caused by the macro spans :-/